.github/workflows/version-packages.yml

```diff
@@ -17,7 +17,6 @@ permissions:
 jobs:
   version:
     runs-on: ubuntu-latest
-    environment: deployment
```

Comment:
This is removed because versioning the packages doesn't need access to any environment variables, right?

Reply:
Right